src/app/modules/OpenshiftStreams/components/StreamsTableConnected/StreamsTableConnected.tsx

```diff
@@ -107,6 +110,32 @@ export const StreamsTableConnected: VoidFunctionComponent<
 
   const [shouldRefresh, setShouldRefresh] = useState(false);
 
+  //kafka size state
+  const [kafkaSize, setKafkaSize] =
+    useState<CreateKafkaInstanceWithSizesTypes.GetSizesData>();
+
+  const fetchKafkaSize = useCallback(async () => {
+    if (
+      !kafkaSize &&
+      kafkaInstancesList?.items &&
+      kafkaInstancesList?.items.length > 0
+    ) {
+      const { cloud_provider, region } =
```

Comment:
This isn't any better than hardcoding the value. What if we have instances from different cloud providers and regions, and for some reason, they have different trial durations?

The right way to do this is to enrich the data used to populate the table to add the size object, similarly to how it's done for the drawer.